webapp/src/pages/IntersectionDashboard.tsx

```diff
@@ -0,0 +1,119 @@
+import {
+  selectIntersections,
+  selectSelectedIntersectionId,
+  setSelectedIntersection,
+} from '../generalSlices/intersectionSlice'
+import MapIconRounded from '@mui/icons-material/Map'
+import MapDialog from '../features/intersections/intersection-selector/intersection-selector-dialog'
+
+function IntersectionDashboard() {
+  const dispatch: ThunkDispatch<RootState, void, AnyAction> = useDispatch()
+  const intersectionId = useSelector(selectSelectedIntersectionId)
+  const intersections = useSelector(selectIntersections)
+  const [openMapDialog, setOpenMapDialog] = useState(false)
+
+  useEffect(() => {
+    dispatch(updateRsuTableData())
+    dispatch(getAvailableUsers())
+  }, [dispatch])
+
+  return (
+    <>
+      <div id="admin">
+        <h2 className="adminHeader">CV Manager Admin Interface</h2>
+        <Tooltip title="Select Intersection">
+          <FormControl sx={{ mt: 1, minWidth: 200 }}>
+            <InputLabel>Intersection ID</InputLabel>
+            <Select
+              value={intersectionId}
+              onChange={(e) => {
+                dispatch(setSelectedIntersection(e.target.value as number))
+              }}
+            >
+              {/* TODO: Update to display intersection Name */}
+              {intersections.map((intersection) => (
+                <MenuItem value={intersection.intersectionID}>{intersection.intersectionID}</MenuItem>
+              ))}
+            </Select>
+          </FormControl>
+        </Tooltip>
+        <Tooltip title="Select Intersection on Map">
+          <IconButton
+            onClick={() => {
+              setOpenMapDialog(true)
+            }}
+          >
+            <MapIconRounded fontSize="large" />
+          </IconButton>
+        </Tooltip>
+        <VerticalTabs
+          notFoundRoute={
+            <NotFound
+              redirectRoute="/dashboard/intersection"
+              redirectRouteName="Intersection Dashboard Page"
+              description="This page does not exist. Please return to the main admin page."
+            />
+          }
+          defaultTabIndex={0}
+          tabs={[
+            {
+              path: 'dashboard',
+              title: 'Dashboard',
+              child: <DashboardPage />,
+            },
+            {
+              path: 'notifications',
+              title: 'Notifications',
+              child: <NotificationPage />,
+            },
+            {
+              path: 'data-selector',
+              title: 'Data Selector',
+              child: <DataSelectorPage />,
+            },
+            // The decoder page is still under development
+            // {
+            //   path: 'decoder',
+            //   title: 'Decoder',
+            //   child: <DecoderPage />,
+            // },
+            {
+              path: 'reports',
+              title: 'Reports',
+              child: <ReportsPage />,
+            },
+            // {
+            //   path: 'configuration',
```

Comment:
Is this commented section also under develop?

Reply:
Yes, the configuration and decoder pages are under active development